integration-tests/README.md

```diff
@@ -311,3 +311,16 @@ ITOperator.java - take a look at this test for reference
 # Future enhancement
 
 Add functional tests
+
+## Troubleshooting
+
+The integration tests are not completely independent of the environment.
+
+You may run into one or more of the following errors when you attempt to execute the command
```

Comment:
attempt to execute the command -> attempt to execute the command:

Reply:
@tbarnes-us Does that work for you?

@rjeberhard Does that work for you?

Reply:
I'm fine with adding a colon, but my opinion is of limited value here:   Rosemary gets final say on all syntactical edits.   :-)

Reply:
sorry... I was asking what you thought of this comment from Rosemary, "I'm seeing so many things that need to be cleaned up that it would be easier for me if I reviewed and edited the files myself after they are merged. I'll make a note of which files I need to look at."

Reply:
oh ... That seems fine to me.

Reply:
done